### DISCUSSION TITLE
composer prefix-namespace issue

### DISCUSSION THREAD

**Discussion by hamzouzou** [cbatista8a/graphapi#4] [General]
Context : 
- php : v7.4.33
- prestashop : 1.7.8.10
- php.ini : memory_limit : 2028M

Hi, 
i followed your tuto of install and i have error when i set **composer prefix-namespace** :

PhpScoper version 0.17.5@f67ae1e

 1228/4091 [▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░]  30%PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 20480 bytes) in /modules/graphapi/vendor/nikic/php-parser/lib/PhpParser/Lexer.php on line 319
Script vendor/bin/php-scoper add-prefix --force handling the prefix-namespace event returned with error code 255

Can you help me ? 

Thank you :)

**Comment by cbatista8a** [comment]
Sorry for responding too late, I have not had problems of this type even with the predefined php parameters in my local environment using Docker, if anyone else has similar problems, please consult the documentation of the "nikic/php-parser" library or open an issue in the respective repository since this is outside the control of the "graphapi" module.
